src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -384,6 +385,7 @@ def load_arguments(self, _):
         c.argument('disable_defender', action='store_true', validator=validate_defender_disable_and_enable_parameters)
         c.argument('enable_defender', action='store_true')
         c.argument('defender_config', validator=validate_defender_config_parameter)
+        c.argument('ssh_key_value', required=False, type=file_type, completer=FilesCompleter(), validator=validate_ssh_key_for_update)
```

Comment:
`required=False` is the default setting, we do not need to explicitly define it

Reply:
fixed, thanks